pkgs/by-name/ra/rapidjson/package.nix

```diff
@@ -58,7 +58,7 @@ stdenv.mkDerivation (finalAttrs: {
   cmakeFlags = [
     (lib.cmakeBool "RAPIDJSON_BUILD_DOC" true)
     (lib.cmakeBool "RAPIDJSON_BUILD_TESTS" true)
-    (lib.cmakeBool "RAPIDJSON_BUILD_EXAMPLES" true)
+    (lib.cmakeBool "RAPIDJSON_BUILD_EXAMPLES" false)
```

Comment:
Doesn't matter much, but add a comment like we do below for cxx11? /shrug

Reply:
I think the reason not to build them is just that they’re useless, and that it breaks with CMake 4 is sort of incidental. I probably wouldn’t flip this switch back on even if they fixed the CMake file upstream and we wired them up to be installed.

Reply:
I guess my only thing is like, "should we say not to enable them," but meh.